.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,6 +12,8 @@ Please complete the following checklist when submitting a PR:
 
 - [ ] Ensure that the test suite passes, by running `make test`.
 
+- [ ] Ensure that code is properly formatted, by running `make format`.
+
```

Comment:
Good catching adding this in! 

I wonder if it is useful to also say:
```suggestion
- [ ] Ensure that code is properly formatted, by running `make format` or `black -l 100 strawberryfields`.
      You will need to have the Black code format installed: pip install black
```

This way, Windows users can run black manually (as they might not have `make` installed), and users that don't have black installed will know what to do.